android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1015,6 +1015,13 @@ public static Typeface getFontTypeface(Context context, String fontFilename) {
 
     try {
       if ((fontFilename != null) && FileUtils.hasFontExtension(fontFilename)) {
+        // Ignore .woff files if Android 7.0 / 7.1 (Issue #4896)
+        if ((Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) &&
+            (Build.VERSION.SDK_INT <= Build.VERSION_CODES.N_MR1) &&
+            fontFilename.toLowerCase().endsWith(FileUtils.WOFFFONT)) {
+          return font;
```

Comment:
I'd prefer `return null;` here to be clear as this is a precondition.

In some ways, I'd prefer to eliminate the `font` variable altogether and simply `return Typeface.createFromFile(file)` on line 1027, with the last line of the method being `return null;` for the error case. I think that is clearer.